webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js

```diff
@@ -2,6 +2,35 @@
 
 $(document).ready(function () {
 
+    /* Idempotent function to display the TelemetryEnvironment with
+     * jQuery JSONView */
+    var displayTelemetryEnvironment = (function() {
+        var once = false;
+        return function inner() {
+            if (once) {
+                // idempotent function
+                return;
+            }
+            once = true;
+            var container = $('#telemetryenvironment-json');
+            if (container.length) {
+                var jsonData = container.data('telemetryenvironment');
+                if (jsonData) {
+                    try {
+                        $('#telemetryenvironment-json').JSONView(jsonData);
+                    } catch (ex) {
+                        console.warn(
+                            "The data in the #telemetryenvironment-json dataset is not valid JSON"
+                        );
+                        container.append(
+                            $('<p>Invalid JSON in TelemetryEnvironment</p>')
+                        );
+                    }
+                }
```

Comment:
You could add an `else` here that spits out "No Telemetry Environment data available."